examples/training.jl

```diff
@@ -48,34 +48,34 @@ function split_validation(X::AbstractVector, Y::AbstractVector, frac=0.1; rng=Ra
     (X[idxs[1:ntrain]], Y[idxs[1:ntrain]]), (X[idxs[ntrain+1:end]], Y[idxs[ntrain+1:end]])
 end
 
-function train!(loss, ps, data, opt, val_data; n_epochs=10)
+function train!(loss, model, train_data, opt, val_data; n_epochs=10)
```

Comment:
Think opt should be called opt_state instead

Reply:
done